README.md

```
@@ -1,70 +1,141 @@
+
+# Install dependencies
+pip install -r requirements.txt
+
+# Run migrations
+python manage.py migrate
+
+# Start server
+python manage.py runserver
+```
+
+3. **Frontend Setup**
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)